src/wp-admin/css/about.css

```diff
@@ -19,25 +19,23 @@
 ------------------------------------------------------------------------------*/
 
 .about__container {
-	/* Section backgrounds */
-	--background: transparent;
-	--subtle-background: #def;
+	/* Page */
+	--about-page--background: #f0f7ff;
 
-	/* Main text color */
-	--text: #000;
-	--text-light: #fff;
+	/* Sections */
+	--about-page--section--background: transparent;
+	--about-page--section-alt--background: #def;
 
+	/* Text colors */
+	--about-page--text--color: var(--text, #000);
+	--about-page--text-light--color: var(--text-light, #fff);
+	
 	/* Accent colors: used in header, on special classes. */
-	--accent-1: #3858e9; /* Accent background, link color */
-	--accent-2: #2d46ba; /* Header background */
+	--about-page--accent-1: var(--accent-1, #3858e9); /* Accent background, link color */
+	--about-page--accent-2: var(--accent-2, #2d46ba); /* Header background */
 
-	/* Navigation colors. */
-	--nav-background: #fff;
-	--nav-border: transparent;
-	--nav-color: var(--text);
-	--nav-current: var(--accent-1);
```

Comment:
These nav values should stick around - the navigation colors don't always map to other colors like this (see [5.7](https://core.trac.wordpress.org/browser/branches/5.7/src/wp-admin/css/about.css#L34), [5.6](https://core.trac.wordpress.org/browser/branches/5.6/src/wp-admin/css/about.css#L42))